src/contexts/apolloContext.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import {AuthContextConsumer} from "./authContext";
+import ApolloClient, {NormalizedCacheObject, InMemoryCache, Operation, gql} from "apollo-boost";
+import ApolloProvider from "react-apollo/ApolloProvider";
+
+const ApiRoot = process.env.REACT_APP_API_URI;
+
+const GET_VIEWER = gql`
+    {
+        viewer {
+            id
+            user
+        }
+    }
+`;
+
+type Props = {}
+
+const apolloContext: React.FunctionComponent<Props> = (props) => {
```

Comment:
IDK if i can get away with making this a stateless functional component. we'll find out!

Reply:
i did it